src/Cli/dotnet/commands/dotnet-test/VSTestFeatureFlag.cs

```diff
@@ -9,35 +9,35 @@ namespace Microsoft.DotNet.Tools.Test
     // !!! FEATURES MUST BE KEPT IN SYNC WITH https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.CoreUtilities/FeatureFlag/FeatureFlag.cs !!!
```

Comment:
It says it must be kept in sync but do you mean like exact copy/paste or having same level of features? We are not implementing `IFeatureFlag` for example.

Also we are not porting the `DISABLE_ARTIFACTS_POSTPROCESSING_NEW_SDK_UX` feature.

Reply:
I meant with flag names where pertinent...not all env vars are used here, we can copy past everything anyway to avoid confusion